app/Nova/Metrics/AttendancePerWeek.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Nova\Metrics;
+
+use App\Attendance;
+use Illuminate\Http\Request;
+use Laravel\Nova\Metrics\Trend;
+
+class AttendancePerWeek extends Trend
+{
+    /**
+     * Calculate the value of the metric.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return mixed
+     */
+    public function calculate(Request $request)
+    {
+        // This is slightly hacky, but it works. Otherwise, responses with a created date of midnight (as created by
+        // some forms) were pushed back to the previous day in the metric. This acts like we're in GMT while
+        // calculating the attendance.
+        $originalTimezone = $request->timezone;
+        $request->timezone = 'Etc/GMT';
+
+        // If we're on a team page, not the main dashboard, filter to that team
+        if ($request->resourceId) {
+            $query = (new Attendance())
+                ->newQuery()
+                ->where('attendable_id', $request->resourceId)
+                ->where('attendable_type', 'App\Team');
+
+            $result = $this->countByWeeks($request, $query)->showLatestValue();
+        } else {
+            $result = $this->countByWeeks($request, Attendance::class)->showLatestValue();
+        }
+
+        $request->timezone = $originalTimezone;
+
+        return $result;
+    }
+
+    /**
+     * Get the ranges available for the metric.
+     *
+     * @return array
+     */
+    public function ranges()
+    {
+        return [
+            4 => '1 Month',
+            8 => '2 Months',
```

Comment:
Can you change the default to 2 months somehow? 4 data points are not particularly interesting.

Reply:
I just removed the 1 month option. As far as I can tell the default is always the first option.